FWTool/FileWaveImporter.py

```diff
@@ -175,6 +179,10 @@ def main(self):
                                      (import_source, fileset_name, e))
 
         finally:
+            model_update = self.env.get('fw_model_update', None)
+            if model_update == True:
+                self.client.model_update()
```

Comment:
This can raise
`subprocess.CalledProcessError`
in case Model update fails (for instance if something is locked by an admin).
Ideally the client code should be updated to raise a proper exception - but I'd add error handling (so you at least unmount DMG if needed even if model update failed)